tests/schemas/api_v3/tokens.schema.js

```diff
@@ -17,7 +17,7 @@ import Joi from 'joi';
 import regex from './regex';
 
 const lockedAmount = {
-	moduleID: Joi.string().pattern(regex.MODULE_COMMAND_ID).required(),
+	module: Joi.string().pattern(regex.MODULE_COMMAND).required(),
```

Comment:
`module` would not satisfy the pattern.
```suggestion
	module: Joi.string().required(),
```

Reply:
use `regex.MODULE` which is similar to `regex.NAME`

Reply:
Applied.